tests/test_env.py

```diff
@@ -314,7 +253,11 @@ def test_process_env_install_from_source(
     package_name: str = "inmanta-module-minimalv2module"
     project_dir: str = os.path.join(modules_v2_dir, "minimalv2module")
     assert package_name not in env.process_env.get_installed_packages()
-    env.process_env.install_from_source([env.LocalPackagePath(path=project_dir, editable=editable)])
+    env.process_env.install_for_config(
+        requirements=[],
+        paths=[env.LocalPackagePath(path=project_dir, editable=editable)],
+        config=PipConfig(use_system_config=True),
```

Comment:
Is this required? I see that it's consistent with what we did before, but I don't think we need it.

Reply:
reverted to test legacy method